eva/utils/kv_cache.py

```diff
@@ -26,10 +26,10 @@ class DiskKVCache:
         `max_cache_size` (int, optional): an integer representing the maximum size of
             the cache. The system will make its best effort to enforce this limit, but it may slightly exceed it. The default value is 2**30.
         `shards` (int, optional): an integer representing the number of shards to use.
-            This can improve concurrent writes. The default value is 6.
+            This can improve concurrent writes. The default value is 3.
     """
 
-    def __init__(self, path: str, max_cache_size: int = 2**30, shards: int = 6):
+    def __init__(self, path: str, max_cache_size: int = 2**30, shards: int = 3):
```

Comment:
This means 3 GB -- can we add a comment in the code itself?

Reply:
Done